apm-agent-plugins/apm-profiling-plugin/pom.xml

```diff
@@ -15,5 +15,25 @@
         <apm-agent-parent.base.dir>${project.basedir}/../..</apm-agent-parent.base.dir>
     </properties>
 
+    <dependencies>
+        <dependency>
+            <groupId>org.kohsuke</groupId>
+            <artifactId>github-api</artifactId>
+            <version>1.133</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-compress</artifactId>
+            <version>1.21</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>tools.profiler</groupId>
+            <artifactId>async-profiler</artifactId>
+            <version>1.8.3</version>
```

Comment:
[minor] might be worth having a build property to define the version, then reuse this in `AsyncProfilerUpgrader` to avoid any confusion. Or, maybe simpler if `AsyncProfilerUpgrader` would be able to extract the version from this dependency at runtimeand remove the `AsyncProfilerUpgrader#TARGET_VERSION`

Reply:
This is the Java API dependency, not the lib itself - you see its version is 1.8.3. We don't need to maintain this unless the API breaks